src/DIPS.Xamarin.UI/Controls/TrendGraph/TrendGraph.xaml.cs

```diff
@@ -0,0 +1,233 @@
+        }
+
+        private static void OnAnyPropertyChanged(BindableObject bindable, object oldvalue, object newvalue)
+        {
+            if (!(bindable is TrendGraph trendGraph)) return;
+            trendGraph.StartRedraw(trendGraph, EventArgs.Empty);
+        }
+
+        /// <summary>
+        ///  <see cref="GraphColor" />
+        /// </summary>
+        public static readonly BindableProperty GraphColorProperty =
+            BindableProperty.Create(nameof(GraphColor), typeof(Color), typeof(TrendGraph), Color.FromHex("#a26eba"), propertyChanged: OnAnyPropertyChanged);
+
+        /// <summary>
+        /// Color of the filled graph
+        /// </summary>
+        public Color GraphColor
+        {
+            get { return (Color)GetValue(GraphColorProperty); }
+            set { SetValue(GraphColorProperty, value); }
+        }
+
+        /// <summary>
+        ///  <see cref="GraphBackgroundColor" />
+        /// </summary>
+        public static readonly BindableProperty GraphBackgroundColorProperty =
+            BindableProperty.Create(nameof(GraphBackgroundColor), typeof(Color), typeof(TrendGraph), Color.FromHex("#edf3f4"), propertyChanged: OnAnyPropertyChanged);
+
+        /// <summary>
+        /// Color behind the graph
+        /// </summary>
+        public Color GraphBackgroundColor
+        {
+            get { return (Color)GetValue(GraphBackgroundColorProperty); }
+            set { SetValue(GraphBackgroundColorProperty, value); }
+        }
+
+        /// <summary>
+        ///  <see cref="MaxValue" />
+        /// </summary>
+        public static readonly BindableProperty MaxValueProperty =
+            BindableProperty.Create(nameof(MaxValue), typeof(double), typeof(TrendGraph), 100.0, propertyChanged: OnAnyPropertyChanged);
+
+        /// <summary>
+        /// Max value of the trend
+        /// </summary>
+        public double MaxValue
+        {
+            get { return (double)GetValue(MaxValueProperty); }
+            set { SetValue(MaxValueProperty, value); }
+        }
+
+        /// <summary>
+        ///  <see cref="MinValue" />
+        /// </summary>
+        public static readonly BindableProperty MinValueProperty =
+            BindableProperty.Create(nameof(MinValue), typeof(double), typeof(TrendGraph), 0.0, propertyChanged: OnAnyPropertyChanged);
+
+        /// <summary>
+        /// Min value of the trend
+        /// </summary>
+        public double MinValue
+        {
+            get { return (double)GetValue(MinValueProperty); }
+            set { SetValue(MinValueProperty, value); }
+        }
+
+        /// <summary>
+        ///  <see cref="GraphMargin" />
+        /// </summary>
+        public static readonly BindableProperty GraphMarginProperty =
+            BindableProperty.Create(nameof(GraphMargin), typeof(double), typeof(TrendGraph), 0.5, propertyChanged: OnAnyPropertyChanged);
+
+        /// <summary>
+        /// Margin between each item
```

Comment:
Do not think we need this yet? I did not add any configurable way of margins between radiobuttons / labels in radiobuttongroup because I want the consumers to get experience with it before adding it.

Reply:
Default of 0.5 looks good.
But the usecase is when you want to set 0 to have the bars next to each other.